apluslms_roman/cli.py

```diff
@@ -197,6 +212,12 @@ def add_cli_actions(parser):
         help=_("build the course (default action)"))
     build.add_argument('course', nargs='?',
         help=_("location of the course definition (default: current working dir)"))
+    build.add_argument('--buildsteps', nargs='+',
```

Comment:
This should include `-s` and long option should be `--steps`. Then in `build_action` we can check if first item of `steps` is a `?`. In addition, build subcommand could have `--list-steps` flag to do the same.